ACEHAL/dyn_utils.py

```diff
@@ -52,6 +52,48 @@ def __call__(self):
             # reject
             atoms.set_cell(orig_cell, True)
 
+class SwapMC:
+    """ASE trajectory attachment that does swap MC steps
+
+    Parameters
+    ----------
+    atoms: Atoms
+        atomic configuration
+    temperature_K: float
+        temperature in K
+    P_GPa: float
+        pressure in GPa
+    """
+    def __init__(self, atoms, temperature_K, P_GPa):
+        self.atoms = atoms
+
+        self.T = temperature_K
+        self.P = P_GPa
```

Comment:
`self.P` no longer used anywhere.  Should we just drop that argument from the constructor?  Or is there a reason to keep it?

Reply:
@casv2 you OK with dropping `self.P`?